src/components/entity/list.vue

```diff
@@ -383,15 +383,12 @@ export default {
         this.snapshotFilter += `(__system/deletedAt eq null or __system/deletedAt gt ${this.now})`;
       }
     },
-    // This method is called directly by DatasetEntities.
+    // This method is called directly by DatasetEntities. + the @reset-click handler of filters
     reset() {
-      if (this.odataFilter == null && !this.searchTerm)
+      if (this.odataFilter == null && !this.searchTerm) {
         this.fetchChunk(true);
```

Comment:
So if all filters are already reset / the default, then pressing the reset button will refresh the table? That seems fine to me, though I think it works slightly differently from `SubmissionList`. In `SubmissionList`, I think nothing would happen in that case.

Idea for alternative: add a `resetFilters()` method similar to `SubmissionList`, then call `resetFilters()` from `reset()` in the `else` case.

```vue
@reset-click="resetFilters">
</template>

<script>
resetFilters() {
  this.$router.replace({ path: this.$route.path, query: {} });
},
reset() {
  if (this.odataFilter == null && !this.searchTerm) {
    this.fetchChunk(true);
  } else {
    this.resetFilters();
  }
}
```